actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -63,22 +63,23 @@ def _filtered_referrer # :nodoc:
       # of an options hash will generate a link to the referrer (a JavaScript back link
       # will be used in place of a referrer if none exists). If +nil+ is passed as the name
       # the value of the link itself will become the name.
+      # Use link_to with a block to wrap the whole block with your link.
       #
       # ==== Signatures
       #
-      #   link_to(body, url, html_options = {})
+      #   link_to(name, url, html_options = {})
       #     # url is a String; you can use URL helpers like
       #     # posts_path
       #
-      #   link_to(body, url_options = {}, html_options = {})
+      #   link_to(name, url_options = {}, html_options = {})
       #     # url_options, except :method, is passed to url_for
       #
       #   link_to(options = {}, html_options = {}) do
-      #     # name
+      #     # name and maybe some HTML
```

Comment:
@senny I went for consistency here and as you said "name" was used more often. From the two choices "name" and "body" it feels better to me. There does not seem to be a clear naming convention (at least in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). "link text" would be more specific, but I don't think its necessary to change it that much, just to make it more consistent inside the url helper docs.